.circleci/config.yml

```diff
@@ -1,7 +1,18 @@
 version: 2.1
+parameters:
+  aws-profile:
+    type: string
+    default: default
+  aws-region:
+    type: string
+    default: us-east-2
+  cluster-name:
+    type: string
+    default: operationcode-backend
```

Comment:
I think all of these parameters are going to need to move down inside the job (currently line 82)